src/Controllers/Database/DataDictionaryController.php

```diff
@@ -110,4 +110,23 @@ public function __invoke(ServerRequest $request): Response
 
         return $this->response->response();
     }
+
+    /**
+     * @param mixed[][] $indexes index data
+     *
+     * @return array<true> The list of primary key columns
```

Comment:
```suggestion
     * @return array<string,true> The list of primary key columns
```
I think documenting the key is important

Reply:
Ok, but then it has to be `array<array-key, true>` or `array<string|int, true>`. Psalm doesn't allow me to make it as `string` only because the key can be either `int` or `string`.